README.md

```diff
@@ -388,6 +388,10 @@ npm run watch
 Many of our Fractal view templates are compatible with [Nunjucks](https://mozilla.github.io/nunjucks/) (for JavaScript/Node), [Jinja](http://jinja.pocoo.org/docs/2.9/) (Python), and [Twig](https://twig.sensiolabs.org/) (PHP) out of the box. Components that reference other components use a Fractal-specific `{% render %}` tag that will either need to be implemented in other environments or replaced with the appropriate `{% include %}` tags.
 
 
+## Longtime support of v1.x
```

Comment:
One more thing: Should we say `Long-term support` per https://en.wikipedia.org/wiki/Long-term_support?

Reply:
Fixed in 6593dd0